package.json

```diff
@@ -21,28 +21,32 @@
 		"access": "public"
 	},
 	"files": [
-		"cjs",
-		"esm"
+		"dist"
 	],
-	"main": "./cjs/index.js",
-	"types": "./cjs/index.d.ts",
-	"module": "./esm/index.js",
+	"type": "module",
+	"main": "./dist/index.js",
+	"types": "./dist/index.d.ts",
 	"sideEffects": false,
+	"imports": {
+		"#root/*": "./src/*"
+	},
+	"exports": {
+		".": "./dist/index.js",
+		"./*/": "./dist/*/index.js",
+		"./*": "./dist/*"
+	},
+	"engines": {
+		"node": ">=20.0.0"
+	},
```

Comment:
what's the rationale behind requiring node 20+ now? node 18 is still in LTS and will be in widespread use

Reply:
No particular reason except lockfile differs if being built with 18 and 20 versions.
As i understand it forces users to have node20+ - ill lower requirement to node18 (as it is lowest for ESM packages)

Reply:
> As i understand it forces users to have node20+

yes this is the trouble we are having while trying to upgrade

> ill lower requirement to node18

much appreciated, thank you!

Reply:
`24.0.2` is live